batched/dense/impl/KokkosBatched_Trsm_Serial_Internal.hpp

```diff
@@ -83,8 +112,8 @@ KOKKOS_INLINE_FUNCTION int SerialTrsmInternalLeftLower<Algo::Trsm::Unblocked>::i
 template <>
 template <typename ScalarType, typename ValueType>
 KOKKOS_INLINE_FUNCTION int SerialTrsmInternalLeftLower<Algo::Trsm::Blocked>::invoke(
-    const bool use_unit_diag, const int m, const int n, const ScalarType alpha, const ValueType *KOKKOS_RESTRICT A,
-    const int as0, const int as1,
+    const bool use_unit_diag, [[maybe_unused]] const bool do_conj, const int m, const int n, const ScalarType alpha,
```

Comment:
```suggestion
    const bool use_unit_diag, const bool /* do_conj */, const int m, const int n, const ScalarType alpha,
```
It looks more like it is never used... in that case just omit the variable name in the signature.